src/pt/pigeons.jl

```diff
@@ -125,12 +125,13 @@ Call [`adapt_tempering()`](@ref) followed by
 [`adapt_explorer`](@ref).
 """
 function adapt(pt, reduced_recorders)
-    updated_tempering = adapt_tempering(pt.shared.tempering, reduced_recorders)
+    updated_tempering = adapt_tempering(pt.shared.tempering, reduced_recorders, pt.shared.iterators, pt.inputs.var_reference)
```

Comment:
seems redundant to add pt.shared.iterators, since the recorders can be queried to find out if enough samples have been collected. But not a big deal.